src/qt/pivx/defaultdialog.cpp

```diff
@@ -35,9 +35,9 @@ DefaultDialog::DefaultDialog(QWidget *parent) :
     ui->btnSave->setText("OK");
     ui->btnSave->setProperty("cssClass", "btn-primary");
 
-    connect(ui->btnEsc, SIGNAL(clicked()), this, SLOT(close()));
-    connect(ui->btnCancel, SIGNAL(clicked()), this, SLOT(close()));
-    connect(ui->btnSave, &QPushButton::clicked, this, &DefaultDialog::accept);
+    connect(ui->btnEsc, &QPushButton::clicked, this, &DefaultDialog::close);
+    connect(ui->btnCancel, &QPushButton::clicked, this, &DefaultDialog::close);
+    connect(ui->btnSave, &QPushButton::clicked, [this](){this->isOk = true; accept();});
```

Comment:
No need to do this, the overloaded `accept` method already sets `isOk` to true.

Reply:
done